README.md

```diff
@@ -50,6 +63,43 @@ resource "ironic_node_v1" "openshift-master-0" {
 }
 ```
 
+## Ports
+
+Ports may be specified as part of the node resource, or as a separate `ironic_port_v1`
+declaration.
+
+```terraform
+resource "ironic_port_v1" "openshift-master-0-port-0" {
+  node_uuid   = "${ironic_node_v1.openshift-master-0.id}"
+  pxe_enabled = true
+  address     = "00:bb:4a:d0:5e:38"
+}
+```
+
+## Allocation
+
+The Allocation resource represents a request to find and allocate a Node
+for deployment. The microversion must be 1.52 or later.
```

Comment:
We should really handle microversions ourselves..

Reply:
To what extent? Should the provider only be valid for a minimum microversion?  Or do I really need to maintain knowledge in this code base of everything that a particular microversion supports?

Reply:
We don't need to know every microversion, but we can assume that if allocations are used, microversion must be at least 1.52. So if a user does not provide a microversion higher than that, use 1.52.